diff --git a/node_modules/isomorphic-dompurify/dist/browser.mjs b/node_modules/isomorphic-dompurify/dist/browser.mjs
index d90d773..2172adb 100644
--- a/node_modules/isomorphic-dompurify/dist/browser.mjs
+++ b/node_modules/isomorphic-dompurify/dist/browser.mjs
@@ -1,15 +1,15 @@
 // src/browser.ts
 import DOMPurify from "dompurify";
 var browser_default = DOMPurify;
-var sanitize = DOMPurify.sanitize.bind(DOMPurify);
+var sanitize = DOMPurify && DOMPurify.sanitize ? DOMPurify.sanitize.bind(DOMPurify) : () => "";
 var isSupported = DOMPurify.isSupported;
-var addHook = DOMPurify.addHook.bind(DOMPurify);
-var removeHook = DOMPurify.removeHook.bind(DOMPurify);
-var removeHooks = DOMPurify.removeHooks.bind(DOMPurify);
-var removeAllHooks = DOMPurify.removeAllHooks.bind(DOMPurify);
-var setConfig = DOMPurify.setConfig.bind(DOMPurify);
-var clearConfig = DOMPurify.clearConfig.bind(DOMPurify);
-var isValidAttribute = DOMPurify.isValidAttribute.bind(DOMPurify);
+var addHook = DOMPurify && DOMPurify.addHook ? DOMPurify.addHook.bind(DOMPurify) : () => {};
+var removeHook = DOMPurify && DOMPurify.removeHook ? DOMPurify.removeHook.bind(DOMPurify) : () => {};
+var removeHooks = DOMPurify && DOMPurify.removeHooks ? DOMPurify.removeHooks.bind(DOMPurify) : () => {};
+var removeAllHooks = DOMPurify && DOMPurify.removeAllHooks ? DOMPurify.removeAllHooks.bind(DOMPurify) : () => {};
+var setConfig = DOMPurify && DOMPurify.setConfig ? DOMPurify.setConfig.bind(DOMPurify) : () => {};
+var clearConfig = DOMPurify && DOMPurify.clearConfig ? DOMPurify.clearConfig.bind(DOMPurify) : () => {};
+var isValidAttribute = DOMPurify && DOMPurify.isValidAttribute ? DOMPurify.isValidAttribute.bind(DOMPurify) : () => false;
 var version = DOMPurify.version;
 var removed = DOMPurify.removed;
 function clearWindow() {
