--- components/TaskBarMenu/index.tsx
+++ components/TaskBarMenu/index.tsx
@@ -296,7 +296,7 @@
                                 >
                                     <div className="flex items-center gap-1">
                                         <IconApps className="size-5 text-black transition-transform group-hover/wm:scale-110" />
-                                        <div className="bg-primary/5 dark:bg-white/10 px-1.5 py-0.5 rounded border border-primary/20 text-[11px] font-bold min-w-[1.25rem] text-center shadow-inner !text-black">
+                                        <div className="bg-primary/5 dark:bg-white/10 px-1.5 py-0.5 rounded-full border border-primary/20 text-[11px] font-bold min-w-[1.25rem] text-center shadow-inner !text-black">
                                             {totalWindows}
                                         </div>
                                     </div>
